analysis/semsim/unifiedmodel.py

```diff
@@ -0,0 +1,84 @@
+from transformers import AutoTokenizer
+import datasets
+
+
+class RRUMDatasetArrow():
+    tokenizer = AutoTokenizer.from_pretrained(
```

Comment:
Move also tokenizer to init and store with self?

Reply:
This way we can share one tokenizer between all datasets, which I think justifies this.  Or am I missing something?